### ISSUE TITLE
Paralyze rune rlly break

### ISSUE THREAD

**Issue by rikitrini** [otland/forgottenserver#1290]
Hello guys well, im there for ask about one bug because the paralyze its not working correctly
so if you fix the paralyze and work's fine with the player work horrible with the monsters and if i can fix it for the monsters work's horrible for the players and i already added the source paralyze fix but nothing guys, its rlly break so i call to @marksamman  and if someone can help with this issue

**Comment by rikitrini** [comment]
@marksamman i try to make later a video for show you the bug exactly

**Comment by rikitrini** [comment]
@marksamman There is a video mate i made its working with player but its not working with the monster u will see there, thanks  https://www.youtube.com/watch?v=-3DCkEO8m3A&feature=youtu.be

**Comment by Jaimen** [comment]
I can confirm this error, i have exactly the same problem

**Comment by rikitrini** [comment]
No one have the same problem @marksamman mark do u checked the video?

**Comment by marksamman** [comment]
No, there hasn't been any progress on this issue. If this issue is important to you, you may place a bounty on it.

**Comment by whitevo** [comment]
i recreated paralyze rune for someone too with the effects like original tibia (including the pvp flag things)
The thing is: You cant use combat condition to add condition_haste. You have to add condition without combat to monster.

If you want script then hit me up in OTLand (im whitevo)

**Comment by EPuncker** [comment]
> i recreated paralyze rune for someone too with the effects like original tibia (including the pvp flag things)
> The thing is: You cant use combat condition to add condition_haste. You have to add condition without combat to monster.
> 
> If you want script then hit me up in OTLand (im whitevo)

do you still have it? I want to test it and implement on tfs if its the case

**Comment by whitevo** [comment]
> > i recreated paralyze rune for someone too with the effects like original tibia (including the pvp flag things)
> > The thing is: You cant use combat condition to add condition_haste. You have to add condition without combat to monster.
> > If you want script then hit me up in OTLand (im whitevo)
> 
> do you still have it? I want to test it and implement on tfs if its the case

Checked around in my script graveyard, didn't find any matches to my searches.
And it has been a while so I have no clue what I did either :D
